orbit/cmd/desktop/desktop.go

```diff
@@ -452,6 +460,9 @@ func main() {
 						if migrationType != constant.MDMMigrationTypeADE {
 							migrateMDMItem.Enable()
 							migrateMDMItem.Show()
+							showMDMMigrator = true
+						} else {
+							showMDMMigrator = false
```

Comment:
Would it make sense to also call Disable and Hide here, just as a good measure or adding a check that if showMDMMigrator != true  then it hides it?